src/material/list/selection-list.spec.ts

```diff
@@ -1191,6 +1191,18 @@ describe('MatSelectionList with forms', () => {
       expect(testComponent.optionInstances.toArray()[1].selected).toBe(true);
     }));
   });
+
+  it('can bind both selected and value at the same time', () => {
+    const fixture =
+        TestBed
+            .configureTestingModule(
```

Comment:
You should be able to drop this call once the component is added to the `declarations`.

Reply:
Done.